htlcswitch/link_test.go

```diff
@@ -770,6 +770,69 @@ func TestLinkForwardMinHTLCPolicyMismatch(t *testing.T) {
 	}
 }
 
+// TestLinkForwardMaxHTLCPolicyMismatch tests that if a node is an intermediate
+// node and receives an HTLC which is _above_ its max HTLC policy then the
+// HTLC will be rejected.
+func TestLinkForwardMaxHTLCPolicyMismatch(t *testing.T) {
+	t.Parallel()
+
+	channels, cleanUp, _, err := createClusterChannels(
+		btcutil.SatoshiPerBitcoin*5,
+		btcutil.SatoshiPerBitcoin*5)
```

Comment:
Nit: Args should be on their own lines if they can't fit, e.g.:
```
createClusterChannels(
    btcutil.SatoshiPerBitcoin*5, btcutil.SatoshiPerBitcoin*5,
)
```
There are a couple others like this throughout the commit.

Reply:
Fixed, although the commit has moved to #2434. Closing this PR now :)